bugsnag-android-core/src/main/java/com/bugsnag/android/AppDataCollector.kt

```diff
@@ -49,12 +64,55 @@ internal class AppDataCollector(
         )
     }
 
+    @SuppressLint("SwitchIntDef")
+    @Suppress("DEPRECATION")
+    private fun getProcessImportance(): String? {
+        try {
+            val appInfo = ActivityManager.RunningAppProcessInfo()
+            if (appInfo.pid == 0) {
+                return null
+            }
+
+            if (VERSION.SDK_INT >= VERSION_CODES.JELLY_BEAN) {
+                ActivityManager.getMyMemoryState(appInfo)
+            } else {
+                activityManager?.runningAppProcesses?.find {
+                    it.pid == Process.myPid()
+                }?.let {
+                    appInfo.importance = it.importance
+                }
```

Comment:
I think it's worth capturing the PID once instead of calling `myPid` for each running app process:

```suggestion
                val expectedPid = Process.myPid()
                activityManager?.runningAppProcesses
                    ?.find { it.pid == expectedPid }
                    ?.let { appInfo.importance = it.importance }
```